standard/lexical-structure.md

```diff
@@ -132,6 +140,12 @@ When several lexical grammar productions match a sequence of characters in a com
 
 > *Example*: The character sequence `//` is processed as the beginning of a single-line comment because that lexical element is longer than a single `/` token. *end example*
 
+The rules of the lexical grammar are ordered top-to-bottom, with the first matching rule determining the recognised token. Implicit rules ([§7.2.3](lexical-structure.md#723-lexical-grammar)) derived from literal strings in the grammar are ordered before the explicit rules. This ordering simplifies the grammar.
```

Comment:
This top-to-bottom ordering is not mentioned in the original grammar but is how lexical grammars usually work – lexical rules are ordered while parser rules form a tree rooted in a start symbol. It is important to mention it once the implicit rules for keywords etc. are introduced.